Porto-Dividers/Template.hbs

```diff
@@ -14,10 +14,10 @@
 </div>
 {{else}}
     {{#equal Settings.DividerType "Classic"}}
-        <hr class="{{Settings.PatternStyle}} {{Settings.Spacements}}" />
```

Comment:
@alejoroman0605 Is `PatternStyle` no longer needed?  If it's not, does the setting need to be removed from other places as well?

Reply:
In this specific case when settings.DividerType is equal to "Classic", PatternStyle was the conflicting parameter and the divider was not displayed, otherwise, it is used.